samples/wifi/radio_test/src/nrf_wifi_radio_test_shell.c

```diff
@@ -171,11 +178,16 @@ static bool check_valid_channel(unsigned char chan_num)
 	    (chan_num == 149) ||
 	    (chan_num == 153) ||
 	    (chan_num == 157) ||
+	    (chan_num == 159) ||
 	    (chan_num == 161) ||
-	    (chan_num == 165) ||
-	    (chan_num == 169) ||
-	    (chan_num == 173) ||
-	    (chan_num == 177)) {
+	    (chan_num == 163) ||
+		(chan_num == 165) ||
```

Comment:
Indentation

Reply:
The new lines added here have tabs at the start of the line. This is causing the indentation issue wrt previous lines which are using spaces.

Reply:
Fixed the difference in indentation.